tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/symbols/TypeSymbolTest.java

```diff
@@ -203,6 +205,35 @@ public Object[][] getTupleTypePosition() {
         };
     }
 
+    @Test(dataProvider = "TypeSymbolLocationPos")
+    public void testTypeSymbolLocation(int line, int col, int expLine, int expSCol, int expECol) {
```

Comment:
Shall we use `testStructuredTypeSymbolLocation` test or any more niche method name